python_scripts/nao_server.py

```diff
@@ -225,7 +226,8 @@ def tutor(self,categ):
                             id = self.goNao.session_intro(int(self.sessionNum))
 
                         #create appropriate session object
-                        self.current_session = Session(pid=self.pid, sessionNum=self.sessionNum)    
+                        # IMPORTANT! Commenting this out temporarily
+                        # self.current_session = Session(pid=self.pid, sessionNum=self.sessionNum)
```

Comment:
@kevjiang Here's the other line that I've commented out. One of the bugs involves the fact that, in profile_models.py, the sessionNum attribute is actually named session_num. Another bug had something to do with the use of the super constructor in profile_models.py.